components/collapsible-panel/test/collapsible-panel.visual-diff.html

```diff
@@ -25,9 +25,11 @@
 			padding: 1rem;
 			width: 600px;
 		}
+		.visual-diff.inline {
+			padding: 0;
+		}
 		.visual-diff.no-overflow {
 			overflow: hidden;
-			padding: 0;
```

Comment:
`overflow: hidden` doesn't work with `sticky`, so just applying it to the `focus` tests that need it to catch a regression from https://github.com/BrightspaceUI/core/pull/3326